lchat.c

```diff
@@ -248,21 +247,18 @@ main(int argc, char *argv[])
 	else
 		printf("\033]0;%s\a", title);
 
-	/* preprate terminal reset on exit */
+	/* prepare terminal reset on exit */
 	if (tcgetattr(fd, &origin_term) == -1)
 		die("tcgetattr:");
 
+	term = origin_term;
+
 	if (atexit(exit_handler) == -1)
 		die("atexit:");
 
-	/* prepare terminal */
-	if (tcgetattr(fd, &term) == -1)
-		die("tcgetattr:");
-
-	/* TODO: clean up this block.  copied from cfmakeraw(3) */
-	term.c_iflag &= ~(IMAXBEL|BRKINT|PARMRK|ISTRIP|INLCR|IGNCR|ICRNL|IXON);
-//        term.c_oflag &= ~OPOST;
-	term.c_lflag &= ~(ECHO|ECHONL|ICANON|IEXTEN);
+	term.c_iflag &= ~(BRKINT|PARMRK|ISTRIP|INLCR|IGNCR|ICRNL|IXON);
+	term.c_oflag &= ~OPOST;
```

Comment:
With this flag, I got strange spacing in front of every printed line:
```
abc
   123
      abc
> 
```

Without it looks normal:

```
abc
123
abc
> 
```

Reply:
unfortunately, I cannot reproduce this, but if it's a problem I'm happy to fix it. Before I remove the wrong flag, you're talking about `OPOST`, right?

Reply:
yes. Its about `OPOST`